src/libutil/util.cc

```diff
@@ -220,7 +220,7 @@ std::string base64Encode(std::string_view s)
 }
 
 
-std::string base64Decode(std::string_view s)
+std::string base64Decode(std::string_view s, bool hideValue)
```

Comment:
is it normal in c++ that the default value only needs to be set in the prototype but not here?

Reply:
Yes. I think it will in fact not let you put it in both places.